packages/camera/camera/README.md

```diff
@@ -119,6 +141,15 @@ class _CameraAppState extends State<CameraApp> {
         return;
       }
       setState(() {});
+    }).catchError((e) {
```

Comment:
```
Running command: "dart run code_excerpt_updater --write-in-place --yaml --no-escape-ng-interpolation ../README.md" in /tmp/cirrus-ci-build/packages/camera/camera/example
Processed 1 Dart/Jade/Markdown files: 1 out of 2 fragments needed updating.
�[31mREADME.md is out of sync with its source excerpts.

If you edited code in README.md directly, you should instead edit the example source files. If you edited source files, run the repository tooling's "update-excerpts" command on this package, and update your PR with the resulting changes.
```
Check out https://github.com/flutter/plugins/pull/5339/files#diff-6c463281036ceb5ca6aeb83f1e11574c52ea1a063c88613afa558454b64a943f for example

Reply:
Good to know! Updated.

Reply:
```
  error - example/lib/readme_full_example.dart:39:20 - Missing parameter type for 'e'. Try adding an explicit type, or remove implicit-dynamic from your analysis options file. - implicit_dynamic_parameter
   info - example/lib/readme_full_example.dart:39:20 - Specify type annotations. - always_specify_types

2 issues found.
```

Reply:
code-excerpt for the win! :)